### PR TITLE
Fix CI

### PR DESCRIPTION
**Pull request by tokatoka**

### CODE REVIEW

File: fuzzers/others/libafl-fuzz/Makefile.toml [hunk 1]
```diff
@@ -110,7 +110,7 @@ script = '''
 # cmplog TODO: AFL_BENCH_UNTIL_CRASH=1 instead of timeout 15s
 AFL_LLVM_CMPLOG=1 AFL_PATH=${AFL_DIR} ${AFL_CC_PATH} ./test/test-cmplog.c -o ./test/out-cmplog
 AFL_CORES=1 timeout 5 ${FUZZER} -Z -l 3 -m 0 -V30 -i ./test/seeds_cmplog -o ./test/output-cmplog -c 0 ./test/out-cmplog || true
-test -n "$( ls ${PROJECT_DIR}/test/output-cmplog/fuzzer_main/hangs/id:0000*)" || {
+test -n "$( ls ${PROJECT_DIR}/test/output-cmplog/fuzzer_main/hangs/id:0000* ${PROJECT_DIR}/test/output-cmplog/fuzzer_main/crashes/id:0000*)" || {
```
Comment (domenukk) [inline fuzzers/others/libafl-fuzz/Makefile.toml:113]: It should find crashes so let's keep it in crashes, no? We should maybe increase the runtime or provide testcases closer to the bug

Comment (tokatoka) [reply fuzzers/others/libafl-fuzz/Makefile.toml:113]: it won't find crashes and that's the whole point of this problem

Comment (tokatoka) [reply fuzzers/others/libafl-fuzz/Makefile.toml:113]: because their vm is shit and idk why the real crash always ends up in hangs folder

Comment (domenukk) [reply fuzzers/others/libafl-fuzz/Makefile.toml:113]: well could be a bug on our end?

Comment (tokatoka) [reply fuzzers/others/libafl-fuzz/Makefile.toml:113]: no because this happens only on ci

**Pull request merged**